<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>214</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="route_man">
       <property name="text">
        <string>Route Management</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_man">
       <property name="text">
        <string>Stop Management</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="busroutedriver">
       <property name="text">
        <string>Bus and Routes Assignment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bus_man">
       <property name="text">
        <string>Bus Management</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="driver_man">
       <property name="text">
        <string>Driver Management</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pass_man">
       <property name="text">
        <string>Passenger Management</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="adidtxtbrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Adminidlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Your ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdmin_Dasboard">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuAdmin_Dasboard"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
